scopes/react/react/templates/react-provider/files/aspect.ts

```diff
@@ -0,0 +1,20 @@
+import { ComponentContext } from '@teambit/generator';
+
+export const aspectFile = (context: ComponentContext) => {
+  const { name, namePascalCase: Name } = context;
+  return {
+    relativePath: `${name}.aspect.ts`,
+    content: `import { Aspect } from '@teambit/harmony';
+
+export type ${Name}Config = {
+  // add configurable variables here
+
+};
+
+export const ${Name} = Aspect.create({
+  id: 'my-org.my-aspect-scope/${name}', // IMPORTANT change these values to your own organization + scope
```

Comment:
@debs-obrien i think we have some params for scope we can use here, no?

Reply:
`ComponentID`

Reply:
I am not aware of @davidfirst having implemented it yet but it would be very helpful to have it

Reply:
it's part of the interface
https://github.com/teambit/bit/blob/master/scopes/generator/generator/component-template.ts#L43